changelog/5197.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Fixed incompatibility of Oracle with the :ref:`sql-tracker-store`, by using a ``Sequence``
```

Comment:
does the `:ref:` thing actually work in here?

Reply:
yes, e.g. Rasa 1.6 to Rasa 1.7 here: https://rasa.com/docs/rasa/changelog/#improvements

Reply:
🙈